gym/envs/toy_text/kellycoinflip.py

```diff
@@ -93,7 +93,7 @@ def __init__(self, initialWealth=25, edgePriorAlpha=7, edgePriorBeta=3, maxWealt
         self.maxRounds = maxRounds
         self.rounds = self.maxRounds
         self.maxWealth = maxWealth
-        self._seed()
+        if reseed: self._seed()
```

Comment:
Yes, but why would anyone call `__init__(reseed=False)` if they're not `_reset()`? I guess it could try to check whether `np_random` exists & reseed if it doesn't, just in case someone does that for some reason.